js/getRegistry.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const symphonyRegistry = '\\Software\\Symphony\\Symphony\\';
+
+var Registry = require('winreg')
```

Comment:
This is another typing to define those var but i´ve changed to the regular typing to avoid confusion.